### ISSUE TITLE
AbortException: Worker was terminated.

### ISSUE THREAD

**Issue by kemsky** [stephanrauh/ngx-extended-pdf-viewer#255]
**Describe the bug**
Rarely I receive crash reports in sentry:
```
AbortException: Worker was terminated.
  at BaseExceptionClosure(/assets/pdf.js:648:29)
  at Object.<anonymous>(/assets/pdf.js:651:2)
  at __w_pdfjs_require__(/assets/pdf.js:52:30)
  at Object.<anonymous>(/assets/pdf.js:129:23)
  at __w_pdfjs_require__(/assets/pdf.js:52:30)
  at ? (/assets/pdf.js:116:18)
  at ? (/assets/pdf.js:119:10)
  at webpackUniversalModuleDefinition(/assets/pdf.js:31:50)
  at ? (/assets/pdf.js:32:3)
```

**Version info**
- Version of ngx-extended-pdf-viewer 2.1.2

**Desktop (please complete the following information):**
 - OS: Windows 10, Chrome OS
 - Browser Chrome 
 - Version 80.0.3987

**To Reproduce**
This happens rarely on different files, looks like something fails in pdf.js. It looks like it happens when popup with pdf viewer is closed.

**Comment by stephanrauh** [comment]
Also see https://github.com/stephanrauh/ngx-extended-pdf-viewer/issues/221. 

I suppose I should add an exception handler. The error message probably describes precisely what's going on: the user has decided to close the modal before the worker thread has finished rendering the PDF file completely. I just don't know how to kill the worker thread silently. Calling `PDFViewerApplication.pdfLoadingTask.destroy()` works, but it raises the error message you've reported.

Any ideas?

**Comment by kemsky** [comment]
@stephanrauh, I was able to generate a bunch of different errors (including this one). You have to turn on Slow 3G in dev tools and then close popup (or destroy component) at different times, at different stages of loading/rendering process.

This particular error can be caught here:

```
    // viewer.js
    const loadingTask = (0, _pdfjsLib.getDocument)(parameters);
    loadingTask.promise.catch(e => {
                console.warn(e, '>>>>>>>>>>>>>>>>>');
    });
    this.pdfLoadingTask = loadingTask;
```

But unfortunately, it is a consequence, when this error is thrown viewer is empty (broken).

You can reproduce it by loading/destroying viewer component with slow internet multiple times (popup is ideal for this purpose). It looks like loading is stopped incorrectly or there should be some delay, and next time viewer is not able to open pdf and throws this error.

**Comment by stephanrauh** [comment]
Sounds like a big step toward solving this nuisance.  Thanks!

**Comment by stephanrauh** [comment]
I've learned how to reproduce the bug. Switching between the options of https://pdfviewer.net/custom-toolbar quickly causes the bug.

**Comment by stephanrauh** [comment]
I believe I've managed to get rid of the message. In any case, I couldn't find any negative effect when the message shows. There are several other messages indicated you shouldn't load a new PDF document when the first document ist still rendering, but this particular message seems to be self-healing. Although I don't understand how that's possible.

The feature is going to land with version 9.0.0-alpha.1.

**Comment by stephanrauh** [comment]
The bugfix has landed with version 9.0.0-alpha.1.
